acceptance_tests/features/edit_respondent_details.feature

```diff
@@ -43,46 +44,23 @@ Feature: As an internal user
 
   @us058-s001
   Scenario: The user is able to edit the email address of a respondent
-    Given the respondent with email "test_respondent@test.com" is enrolled
-    And the internal user has found the respondents details
+    Given the respondent with email "test_respondent2@test.com" is enrolled
+    And the internal user has found the second respondents details
```

Comment:
So originally the `test_respondent@test.com` that was being used here was actually the `example@example.com` user and when it's email was changed it became unverified and this was causing failures for me.
There's now two new users created for these tests, one to change other details, and one to change it's email. I should probably reword this though because it doesn't really make sense from a business point of view